Loadable.svelte

```diff
@@ -127,15 +130,18 @@
     state = STATES.SUCCESS
     component = LOADED.get(loader)
   } else {
-    onMount(async () => {
-      await load()
-      dispatch('load')
-      if (unloader) {
-        return () => {
-          LOADED.delete(loader)
-          if (typeof unloader === 'function') {
-            unloader()
-          }
+    onMount(() => {
```

Comment:
This was a great catch, the onDestroy handle wasn't being executed.

Reply:
My pleasure!